packages/react-lowcode/src/codegen/ast/node.ts

```diff
@@ -1,15 +1,26 @@
 import ts from "typescript";
 
 export const isImportDeclarationWithName = (node: ts.Node, name: string): boolean | undefined => {
-    if (ts.isImportDeclaration(node)) {
-        if (node.importClause) {
-            if (node.importClause.name) {
-                if (node.importClause.name.escapedText === name) {
-                    return true;
-                }
-            }
+  if (ts.isImportDeclaration(node)) {
+    if (node.importClause) {
+      //handles "import File from './File'" import types
+      if (node.importClause.name) {

```

Comment:
Please minimize too many branches / `ifs` (for readability)

those two can be simplified like this: `if (node.importClause.name?.escapedText === name`